beam-postgres/Database/Beam/Postgres/Syntax.hs

```diff
@@ -417,6 +417,7 @@ instance IsSql92DataTypeSyntax PgDataTypeSyntax where
   dateType = PgDataTypeSyntax (PgDataTypeDescrOid (Pg.typoid Pg.date) Nothing) (emit "DATE")
   timeType prec withTz = PgDataTypeSyntax (PgDataTypeDescrOid (Pg.typoid Pg.time) Nothing) (emit "TIME" <> pgOptPrec prec <> if withTz then emit " WITH TIME ZONE" else mempty)
   timestampType prec withTz = PgDataTypeSyntax (PgDataTypeDescrOid (Pg.typoid Pg.timestamp) Nothing) (emit "TIMESTAMP" <> pgOptPrec prec <> if withTz then emit " WITH TIME ZONE" else mempty)
+  uuidType = PgDataTypeSyntax(PgDataTypeDescrOid (Pg.typoid Pg.uuid) Nothing) (emit "uuid")
```

Comment:
The one thing I'm not certain about with this is that it requires adding the 'uuid-ossp' extension to make the "uuid" type available. This extension has been part of the standard postgres installation for a number of years, but you have to run `CREATE EXTENSION` for it explicitly for the schema where it is needed. Is there any mechanism whereby one could ensure that the extension has been enabled before attempting to create a UUID column?